### PR TITLE
Feature users can see restaurants menus

### PR DESCRIPTION
**Pull request by OleksaRiabukha**
Admins are able to add, edit and delete menu items for the each restaurant. Users can see available menu items of the each restaurant.

### CODE REVIEW

File: app/views/admin/restaurants/show.html.slim [hunk 1]
```diff
@@ -8,5 +9,23 @@
...
     | Link to website: #{@restaurant.website_url}
-= link_to fa_icon('edit'), edit_admin_restaurant_path(@restaurant), class: "btn btn-primary"
-= link_to fa_icon('trash'), admin_restaurant_path(@restaurant), method: :delete, data: { confirm: "Are you sure?" }, class: "btn btn-danger"
\ No newline at end of file
+    p
+    = link_to fa_icon('edit'), edit_admin_restaurant_path(@restaurant), class: "btn btn-primary", style: "margin-right: 2px"
+    = link_to fa_icon('trash'), admin_restaurant_path(@restaurant), method: :delete, data: { confirm: "Are you sure?" }, class: "btn btn-danger", style: "margin-right: 2px"
+    = link_to "Add Dish", new_admin_restaurant_menu_item_path(@restaurant), class: 'btn btn-success'
+
+- if @restaurant.menu_items.any?
```
Comment (BohdanBusko) [inline app/views/admin/restaurants/show.html.slim]: use `presence?` instead of `any?`

File: app/views/admin/restaurants/show.html.slim [hunk 2]
```diff
@@ -8,5 +9,23 @@
...
+  h3 Your Dishes: 
+  .card.md-3 style="margin-top: 10px;"
```
Comment (BohdanBusko) [inline app/views/admin/restaurants/show.html.slim]: try to not use inline styles

File: app/views/restaurants/show.html.slim [hunk 3]
```diff
@@ -11,5 +11,24 @@
     = @restaurant.phone_number
     br
     = @restaurant.website_url
-= link_to "Menu", "#", class: "btn btn-primary"
-= link_to "Go Back", restaurants_path, class: "btn btn-danger"
+    br
+    = link_to "Go Back", restaurants_path, class: "btn btn-danger", style: "margin-top: 5px;"
+
+- if @restaurant.menu_items.available.any?
```
Comment (BohdanBusko) [inline app/views/restaurants/show.html.slim]: from this line till the end move it right

File: db/seeds.rb [hunk 4]
```diff
@@ -10,5 +10,9 @@
 Restaurant.destroy_all
 
 FactoryBot.create_list(:user, 2)
-FactoryBot.create_list(:restaurant, 2, active: true)
 FactoryBot.create_list(:restaurant, 2, active: false)
+
+FactoryBot.create_list(:restaurant, 2, active: true) do |restaurant|
```
Comment (BohdanBusko) [inline db/seeds.rb]: where is `each`? :)

File: spec/requests/admin/menu_items_spec.rb [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+RSpec.describe 'Admin::MenuItems', type: :request do
+  let(:restaurant) { create(:restaurant) }
+
+  context 'when logged in admin tries to' do
+    login_admin
+
+    describe 'GET /admin/restaurants/:id/menu_item/:id' do
+      context 'access existing menu item page' do
+        let(:menu_item) { create(:menu_item, restaurant: restaurant) }
```
Comment (BohdanBusko) [inline spec/requests/admin/menu_items_spec.rb]: create one menu item and reuse it

File: db/seeds.rb (hunk 4 above)
Comment (BohdanBusko) [reply db/seeds.rb]: because as I remember this method returns an array and to go through the array we need `each`

Comment (OleksaRiabukha) [reply db/seeds.rb]: Add, but it worked fine without each))))

**Review by BohdanBusko — APPROVED** [review]

**Pull request merged**